generic/tests/fillup_disk.py

```diff
@@ -44,17 +44,21 @@ def run(test, params, env):
             number += 1
     finally:
         error.context("Cleaning the temporary files...", logging.info)
-        while number >= 0:
-            cmd = "rm -f /%s/fillup.%d" % (fill_dir, number)
-            logging.debug(cmd)
-            s, o = session2.cmd_status_output(cmd)
+        try:
+            clean_cmd = params.get("clean_cmd") % fill_dir
+            logging.debug(clean_cmd)
```

Comment:
The clean_cmd should already saved in debug log start with "Sending command xxxx". So this line is not needed.